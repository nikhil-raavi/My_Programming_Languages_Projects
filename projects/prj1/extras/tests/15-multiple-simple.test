# this file contains multiple simple declarations

var _some_var1 : number;

var _some_2OtherVar23 :
  string;
  