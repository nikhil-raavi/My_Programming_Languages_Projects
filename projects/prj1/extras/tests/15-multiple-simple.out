[
   [
      "_some_var1",
      "number"
   ],
   [
      "_some_2OtherVar23",
      "string"
   ]
]
